pajbot/bot.py

```diff
@@ -666,12 +666,29 @@ def send_message_to_user(self, user, message, event, method="say"):
             self.me(message)
         elif method == "reply":
             if event.type in ["action", "pubmsg"]:
-                self.say(f"@{user.name}, {message}")
+                msg_id = next(tag["value"] for tag in event.tags if tag["key"] == "id")
+                self.reply(msg_id, message)
             elif event.type == "whisper":
                 self.whisper(user, message)
         else:
             log.warning("Unknown send_message method: %s", method)
 
+    def reply(self, msg_id, message, channel=None):
```

Comment:
I have tested that replying still works even if msg_id is empty or msg_id is invalid (e.g. `xD`), it just won't reply.
Since it doesn't reply, and doesn't add the username at the front, it's important that we run the `utils.clean_up_message` function (which you do) :+1: